packages/api-sdk/src/medical/models/network.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+
+export const networkSchema = z.object({
+  name: z.string().optional(),
+  oid: z.string().optional(),
+  state: z.string().optional(),
+  zip: z.string().optional(),
+  rootOrganization: z.string().optional(),
+  managingOrgOid: z.string().optional(),
+});
```

Comment:
I suggest we skip the zod schemas for the SDK and just define the type in pure TS.